integration-tests/src/test/scala/IntegrationTests.scala

```diff
@@ -2,20 +2,31 @@ package integration
 
 import com.squareup.okhttp._
 import org.scalatest.concurrent.{Eventually, IntegrationPatience}
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
 import integration.services.{Config, GuHttp, TestAtomJsonGenerator}
 import java.time.Instant
 import java.util.UUID
 
 import play.api.libs.json.Json
 
-class IntegrationTests extends FlatSpec with Matchers with Eventually with IntegrationPatience with GuHttp with TestAtomJsonGenerator {
+class IntegrationTests extends FlatSpec with Matchers with Eventually with IntegrationPatience with GuHttp with TestAtomJsonGenerator with BeforeAndAfterAll {
 
   val targetBaseUrl: String = Config.targetBaseUrl
   val JSON = MediaType.parse("application/json; charset=utf-8")
 
   def apiUri(atomId: String): String = s"$targetBaseUrl/api/atom/$atomId"
 
+  var createdAtoms: List[String] = List() /* Add all created atoms IDs to this list as first action after atom created. This allows for test cleanup outside the test flow  */
```

Comment:
I appreciate that the use of a var here is very naughty, and I promise not to do it again